TOC.md

```diff
@@ -287,6 +288,7 @@
       + [`CREATE [GLOBAL|SESSION] BINDING`](/sql-statements/sql-statement-create-binding.md)
       + [`CREATE DATABASE`](/sql-statements/sql-statement-create-database.md)
       + [`CREATE INDEX`](/sql-statements/sql-statement-create-index.md)
```

Comment:
Some newly added documents are missing in the TOC. They are added in https://github.com/pingcap/docs/pull/6766.